app/views/products/_comments.html.erb

```diff
@@ -10,7 +10,7 @@
       <% end %>
       <% unless current_user.nil? %>
         <% if signed_in? && (current_user.admin? || current_user.moderator?) %>
-          <%=  render partial:'comment' %>
+          <%=  render partial:"comments/comment", locals: { comment: comment } %>
```

Comment:
Why do we need to use locals on Heroku? 
It's funny that I need to use locals remotely and on local you don't need 'locals: {}' to make it work ^^